.github/assets/ray.yaml

```diff
@@ -0,0 +1,65 @@
+cluster_name: performance-comparisons
+
+provider:
+  type: aws
+  region: us-west-2
+  cache_stopped_nodes: true
+  security_group:
+    GroupName: ray-autoscaler-c1
+
+auth:
+  ssh_user: ubuntu
+  ssh_private_key: ~/.ssh/ci-github-actions-ray-cluster-key.pem
+
+max_workers: 2
+available_node_types:
+  ray.head.default:
+    resources: {"CPU": 0}
+    node_config:
+      KeyName: ci-github-actions-ray-cluster-key
+      InstanceType: i3.2xlarge
```

Comment:
I haven't used graviton processors just yet since I've been focusing on x86 builds for the time being. I can try testing that out soon, however.

That'll just require running on `buildjet-arm` instances instead.